apps/project-renderer/src/components/app/index.js

```diff
@@ -119,6 +105,16 @@ const App = ( {
 		);
 	};
 
+	useStylesheet( '/ui/stable/theme-compatibility/crowdsignal.css' );
```

Comment:
Left this intentionally commented as a way to see the 'Base Theme'. Need to uncomment this before merging.